terraform/environments/nomis/modules/nomis_stack/ec2_database.tf

```diff
@@ -36,6 +15,27 @@ resource "aws_security_group" "database_server" {
   )
 }
 
+resource "aws_security_group_rule" "weblogic_server" {
+  description       = "DB access from weblogic (private subnet)"
+  type              = "ingress"
+  security_group_id = aws_security_group.database_server.id
+  from_port         = "1521"
+  to_port           = "1521"
+  protocol          = "TCP"
+  cidr_blocks       = ["${aws_instance.weblogic_server.private_ip}/32"]
+  depends_on        = [aws_instance.weblogic_server]
```

Comment:
don't think you really need depends_on here.  It should be able to work out since its using the ip address from the weblogic

Reply:
Yeah you're right